company_website/views.py

```diff
@@ -16,6 +20,7 @@
 from company_website.view_helpers import CustomTemplateView
 from company_website.view_helpers import GoogleAdsMixin
 from company_website.view_helpers import add_meta_tags_to_page_context
+from project_liberation.settings.base import STATIC_ROOT
```

Comment:
It wouldn't work on production because you are importing settings directly from one of a settings files. 
You should import them from django settings mechanism:

`from django.conf import settings`
and then in code use settings
`settiings.STATIC_ROOT`

We can never do this that way because we would import only variables from base.py which are override in local_settings.py in productionn